dashboard/config/linux/bits/filesystems.yml

```diff
@@ -57,8 +57,7 @@ config:
  - XFS_POSIX_ACL
  - XFS_RT
  - XFS_SUPPORT_V4: [v5.10]
- - XFS_DEBUG
- - XFS_ASSERT_FATAL
+ - XFS_DEBUG: n
```

Comment:
I would add a comment saying that it enables the image checks rather than code correctness checks.